ykrt/src/thread_intercept.rs

```diff
@@ -0,0 +1,12 @@
+use std::os::raw::{c_void, c_int};
+
+#[no_mangle]
+pub extern "C" fn __wrap_pthread_create(
+    _: *mut c_void,
+    _: *const c_void,
+    _: extern "C" fn(*mut c_void) -> *mut c_void,
+    _: *mut c_void,
+) -> c_int {
+    eprintln!("Intercepted pthread_create call. No support for threads yet!");
+    panic!();
```

Comment:
You can squash these two lines together, by moving the error string inside the `panic`.

Reply:
done 👉 f86bb075b9ae579829d3cfc6023ed2c4ada8331f

Reply:
> In one of my previous PRs, I got a comment to not put the error string inside panic and use eprintln and panic separately, but I don't remember the reason and can't find that PR

That would be a very surprising comment and thankfully I don't remember seeing it!